ubuntudesign/vanilla-framework
# Rename Grid list to Matrix

## Done

- Rename `.p-grid-list` to matrix
- Rewrote all styling to make it responsive and ensure the pattern no longer relies on grid classes or `.last-row / .last-col` classes
- Amended grid to remove margin-bottom of directly descendant children of `.row` and `.col-*` to prevent double bottom margin bug

## QA

- Check code
- Sync with vf.io and check pattern/matrix across breakpoints
- Check against demo - https://vf-io-7e29b.firebaseapp.com/patterns/matrix/

## Details

Fixes #631 #625
